application/extractor/base.go

```diff
@@ -19,18 +19,49 @@ package extractor
 import (
 	"github.com/insolar/insolar/insolar"
 	"github.com/insolar/insolar/logicrunner/builtin/foundation"
+
 	"github.com/pkg/errors"
 )
 
+func Generic(data []byte, returns ...interface{}) error {
```

Comment:
Please give this procedure a more descriptive name. Also it would be nice to have a little godoc for it.